src/Campaigns/Blocks/CampaignDonors/index.ts

```diff
@@ -0,0 +1,14 @@
+import {paragraph as icon} from '@wordpress/icons';
+import metadata from './block.json';
+import Edit from './edit';
+import initBlock from '../shared/utils/init-block';
+
+const {name} = metadata;
+
+export {metadata, name};
+export const settings = {
+    icon,
+    edit: Edit,
+};
+
+export const init = () => initBlock({name, metadata, settings});
```

Comment:
I'm not sure we need this init-block functionality. We can just include the block root file inside blocks.ts.  Like I did it here https://github.com/impress-org/givewp/blob/674531f04a99386c5af072e6a9b17f1698a2029b/src/Campaigns/Blocks/CampaignCover/index.tsx

Reply:
That's the approach Gutenberg uses to register their blocks. What I like on it is that we do not need to think too much on all the pieces necessary to register the block.

Reply:
So it is easier to add complexity to handle block registration (which we already have - registerBlockType), define an object using this function, and then import it somewhere else, and also add it to another list that handle the registration, than just importing the file? 
What is the benefit of having this?

Reply:
We can move forward here and address this another time. I just don't like this approach of abstracting the core functionality without any real benefit. It adds unnecessary complexity so I need to figure out how to register a block, and I already know how to do that.

Reply:
Refactored on 0ac8456179c6abc8786eb6e0c7fc0b47ccc23d10

Reply:
Hmm, I think I wasn't clear enough. We don't need getBlock().forEach functionality. What I'm proposing here is that each block registers the block. Example https://github.com/impress-org/givewp/blob/674531f04a99386c5af072e6a9b17f1698a2029b/src/Campaigns/Blocks/CampaignCover/index.tsx

Reply:
Okay, lets move this bad boy to QA